static/js/util/profile_edit.js

```diff
@@ -62,13 +63,17 @@ export function boundRadioGroupField(keySet: string[], label: string, options: O
         labelStyle={styles.labelStyle}
         value={obj.value}
         label={label}
+        aria-labelledby={labelId}
       />
     );
   });
 
   const value = String(_.get(profile, keySet));
   return (
-    <div className={validationErrorSelector(errors, keySet)}>
+    <div
+      role="group"
```

Comment:
I think `radiogroup` would be better here. Could you also add a `aria-labelledby` attribute here that points to the `profile-radio-group-label` just below? That will make screen readers will annoucne this section as "Gender Radio Button Group".